src/main/sensors/sonar.c

```diff
@@ -142,4 +142,19 @@ int32_t sonarGetLatestAltitude(void)
     return calculatedAltitude;
 }
 
+/**
+ * returns the maximum distance readable by the sonar
```

Comment:
Would be nice to note units of the return type here (I believe it's centimetres).